Installation/CHANGES.md

```diff
@@ -64,9 +64,9 @@ Release date: March 2019
 
 ### 2D and 3D Mesh Generation
 -   Added 3 functions for writing in modern VTK formats:
-    -`CGAL::output_to_vtu()`
-    -`CGAL::write_VTU()`
-    -`CGAL::write_VTP()`
+    -`CGAL::write_C3T3_to_VTU()`
```

Comment:
Remove "modern" from the text. VTK XML formats date from VTK 4.0. It is more than 15 years old (2001-08-22).

If you want to qualify the format, you can say "in VTK XML formats".